x-pack/plugins/observability_solution/apm/public/context/entity_manager_context/entity_manager_context.tsx

```diff
@@ -64,8 +62,17 @@ export function EntityManagerEnablementContextProvider({
     true
   );
 
+  const {
+    data = { enabled: false },
+    status,
+    refetch,
+  } = useFetcher(
```

Comment:
hmmm that's interesting. We use the useFetcher all around. 

And I deleted the `useEntityManager` in favour of using useFetcher because we don't need to manually create all the state manipulation.

Reply:
It's used all around probably in already lazy-loaded chunks of code, I didn't verify if this entity_manager_context module is used in some lazy loaded modules, it just smelled this was increasing the bundle :) You could try restoring the useEntityManager hook and see if the bundle size goes back into the limits

Reply:
@cauemarcondes try https://github.com/elastic/kibana/blob/5ce482df7a067745ea0041aee9f336588cb02028/x-pack/packages/observability/observability_utils/observability_utils_browser/hooks/use_abortable_async.ts#L22-L23, it's a little more lightweight than useFetcher

Reply:
I reverted [it](https://github.com/elastic/kibana/pull/192349#discussion_r1756818919), won't have time to look into it now.